subprojects/files/src/test/groovy/org/gradle/api/internal/file/collections/DirectoryFileTreeTest.groovy

```diff
@@ -142,10 +155,15 @@ class DirectoryFileTreeTest extends AbstractProjectBuilderSpec {
 
         then:
         visitor.assertExpectations()
+
+        where:
+        visitor << visitors
     }
 
-    def visitorCanStopVisit() {
+    def "visitor can stop visit - isReproducible: true"() {
```

Comment:
this needs a given order, so only reproducible makes sense here

Reply:
Can you add the comment in the source code as well?

Reply:
done